git_downloader.py

```diff
@@ -58,7 +60,7 @@ def download_files(args, g, repo_gen):
                 except Exception:
                     logging.exception('Error downloading %s.' % file)
         except Exception:
-             logging.exception('Error fetching repository %s.' % repo.full_name)
```

Comment:
if there is a problem with the name, can you use the repository id instead? (unless the id has the same problem as the full name)